share/cmake/modules/FindExtPackages.cmake

```diff
@@ -104,13 +104,38 @@ if(OCIO_BUILD_PYTHON OR OCIO_BUILD_DOCS)
     endif()
 endif()
 
-# The presence of OpenImageIO allows additional ocio apps and the OSL
+# The presence of OpenImageIO allows additional OCIO apps and the OSL
 # translation unit tests to be built.
 
-# OpenImageIO
-# https://github.com/OpenImageIO/oiio
-if(OCIO_USE_OIIO_CMAKE_CONFIG)
-    find_package(OpenImageIO 2.1.9 CONFIG)
-else()
-    find_package(OpenImageIO 2.1.9)
+if(OCIO_BUILD_APPS OR OCIO_BUILD_TESTS)
+    # OpenImageIO
+    # https://github.com/OpenImageIO/oiio
+
+    set(OIIO_VERSION "2.1.9")
+
+    if(OCIO_USE_OIIO_CMAKE_CONFIG)
```

Comment:
You should also take the `OCIO_USE_OIIO_CMAKE_CONFIG` cmake variable from `main` (i.e. [CMakeLists.txt line 152](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/main/CMakeLists.txt#L152) ).